Frontend/style.css

```diff
@@ -0,0 +1,64 @@
+* {
```

Comment:
name this css file as scrolling.css, lets separate each of the css files, and use style.css for the main css file.

Reply:
done